docassemble/ALAutomatedTestingTests/data/questions/all_tests.yml

```diff
@@ -22,18 +22,14 @@ comment: |
   - [ ] Text input (type date)
   
   ## Showif:
-  - [ ] Radio (yes/no)
-  - [ ] Radio (other)
-  - [ ] Checkbox (none of)
-  - [ ] Checkbox (yes/no)
-  - [ ] Checkbox (other)
-  - [ ] Buttons (yes/no)
```

Comment:
Some of the unchecked items from this list disappeared in the checked items list (e.g., buttons, date text input). Is that because these items never show up as show-ifs?

Reply:
Actually, I think the comment `No showifs: yesnomaybe: (buttons), buttons: (other), continue button field:` addresses my question for buttons, although still not for date text input.

Reply:
The date text input DOM is the same as a regular text input. Maybe later we'll decide to add it for other reasons. Not sure what those would be right now.